dockerfiles/proxy/Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG OPENRESTY_VERSION=1.17.8.2-4-buster-fat
+FROM openresty/openresty:${OPENRESTY_VERSION}
+
+ARG LUA_RESTY_OPENIDC_VERSION=1.7.3.1
+RUN opm get zmartzone/lua-resty-openidc=${LUA_RESTY_OPENIDC_VERSION}
+
+ENV NGINX_LISTEN=80
+ENV APP_LISTEN=127.0.0.1:8000
+ENV OIDC_REDIRECT_URI=/oidc/callback
+ENV OIDC_LOGOUT_PATH=/logout
```

Comment:
Are these pulled in as part of the `ngx.var.*` variables? They match the naming but the casing is different.

Reply:
These are inserted into the nginx config at https://github.com/mozilla/glam/blob/c4fb0bdc3ae93b1518d5dd28f7be2eb1838c224e/dockerfiles/proxy/default.conf.template#L61-L62 using `envsubst`, which then is accessed in the lua code via the `ngx.var.*` vars.